src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -4,58 +4,71 @@
     "origin.toomanyorigins.hare.name": "Hare",
 
     "origin.toomanyorigins.hisskin.name": "Hiss-kin",
-    "origin.toomanyorigins.hisskin.description": "Creeper'ın bu uzak akrabası patlamalarını kaçış taktiği olarak kullanır.",
+    "origin.toomanyorigins.hisskin.description": "Creeperın bu uzak akrabası, patlamalarını kaçış amaçlı kullanır.",
   
     "origin.toomanyorigins.swarm.name": "Swarm",
-    "origin.toomanyorigins.swarm.description": "Kovandan bağımsız olan bu böcekler kavgadan uzak duran uzman çiftçilerdir.",
+    "origin.toomanyorigins.swarm.description": "Swarm ırkı, çatışmadan uzak duran uzman çiftçi ve kovandan bağımsız böceklerdir.",
 
     "origin.toomanyorigins.undead.name": "Undead",
 
     "origin.toomanyorigins.withered.name": "Withered",
+    "origin.toomanyorigins.withered.description": "Withered ırkı, atalarının Wither'ın kaotik gücünden yararlanmalarının bir sonucu olarak öldürdüklerinin yaşam enerjisini emerler.",
   
-    "power.toomanyorigins.overheat.name": "Overheat",
-    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir objeden uzağa tepebilirsin. Bu işlem size hasar verir.",
+    "power.toomanyorigins.overheat.name": "Aşırı Isınma",
+    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir nesneden uzağa doğru roket zıplaması gerçekleştirebilirsin, bu süreç seni biraz incitir.",
     "power.toomanyorigins.aerial_stability.name": "Hava Stabilliği",
     "power.toomanyorigins.aerial_stability.description": "Aşağıya doğru hızınız biraz azaltılmış.",
     "power.toomanyorigins.conductive.name": "İletken",
-    "power.toomanyorigins.conductive.description": "Yıldırım çarptıpında şarjlı olursunuz.",
+    "power.toomanyorigins.conductive.description": "Yıldırım tarafından çarpıldığında şarjlanırsın.",
     "power.toomanyorigins.stubby_legs.name": "Güdük Bacaklar",
-    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların ayaktayken yavaş hareket etmeni sağlar.",
+    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların ayakta daha yavaş hareket etmeni sağlar.",
   
     "effect.toomanyorigins.charged": "Şarjlı",
     "death.attack.overheat": "%1$s içini fitilledi",
     "death.attack.overheat.player": "%1$s, %2$s den kaçmaya çalışırken içini fitilledi.",
     "death.attack.overheat.player.item": "%1$s, %3$s kullanan %2$s den kaçmaya çalışırken içini fitilledi.",
   
-    "power.toomanyorigins.hover.name": "Hover",
-    "power.toomanyorigins.hover.description": "Düşerken süzülebilirsiniz.",
-    "power.toomanyorigins.pollination.name": "Polenlendirme",
-    "power.toomanyorigins.pollination.description": "Eğilmiyorken kemik tozu kullanmak ayrıca birleşik 4 bloğada etki eder.",
-    "power.toomanyorigins.calming_aura.name": "Sakinleştirici Aura",
+    "power.toomanyorigins.hover.name": "Süzül",
+    "power.toomanyorigins.hover.description": "Düşerken yükseklik kaybetmeden süzülebilirsiniz.",
```

Comment:
translated as "Gliding without losing altitude"

and changed the name to just "Glide"